util/usernames

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
Agree. Good motivation.

On Tue, Feb 23, 2016, 9:30 PM Liam <notifications@github.com> wrote:

> In util/usernames
> <https://github.com/tenex/github-contributions/pull/72#discussion_r53885965>
> :
>
> > @@ -0,0 +1,31 @@
> > +#!/usr/bin/env python3
>
> with hashbang + chmod +x, it's not necessary unless you're trying to use
> it as a python library. I like it this way because I don't like it when
> command line utilities (like this one) expose what language they're written
> in via their filename. Agree/disagree?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tenex/github-contributions/pull/72/files#r53885965>.
>